admin/class-automation.php

```diff
@@ -108,6 +128,25 @@ public static function action__admin_menu(): void {
 		);
 	}
 
+	public static function action__admin_print_scripts(): void {
+		wp_enqueue_script(
+			'apple-news-plugin-admin-settings',
+			plugins_url( 'build/adminSettings.js', __DIR__ ),
+			[ 'wp-block-editor', 'wp-api-fetch', 'wp-api', 'wp-i18n', 'wp-components', 'wp-element', 'wp-tinymce' ],
+			[],
+			true
+		);
+		wp_enqueue_style( 'wp-edit-blocks' );
+		wp_localize_script(
+			'apple-news-plugin-admin-settings',
+			'wpLocalizedData',
+			[
+				'taxonomies' => get_taxonomies( [ 'public' => 'true' ] ),
+				'fields'     => self::FIELDS,
```

Comment:
You're probably already doing this, but when you add the fields for section and theme you can populate the available options in `self::FIELDS` based on the sections from the API and the loaded themes.